style.css

```diff
@@ -0,0 +1,456 @@
+body {
+	font-family: 'Gotham Pro';
+	font-size: 14px;
+	max-width: 1030px;
+	min-width: 320px;
+	margin: 0 auto;
+	padding: 32.46px 0;
+}
+
+a {
+	text-decoration: none;
+	color: #000000;
+	text-transform: uppercase;
+}
+
+a:hover {
+	opacity: .5;
+}
+
+/*<header class='header'>*/
+.header {
+	color: #000000;
+	font-size: 12px;
+	font-weight: bold;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ProjectMod2_EKh/pullRequest?prid=6247910)